stdlib/public/core/StringInterpolation.swift

```diff
@@ -197,6 +197,132 @@ public struct DefaultStringInterpolation: StringInterpolationProtocol, Sendable
   }
 }
 
+extension DefaultStringInterpolation {
+  /// Interpolates the given optional value's textual representation, or the
+  /// specified default string, into the string literal being created.
+  ///
+  /// You don't need to call this method directly. It's used by the compiler
+  /// when interpreting string interpolations where you provide a `default`
+  /// parameter. For example, the following code implicitly calls this method,
+  /// using the value of the `default` parameter when `value` is `nil`:
+  ///
+  ///     var age: Int? = 48
+  ///     print("Your age is \(age, default: "unknown")")
+  ///     // Prints: Your age is 48
+  ///     age = nil
+  ///     print("Your age is \(age, default: "unknown")")
+  ///     // Prints: Your age is unknown
+  ///
+  /// - Parameters:
+  ///   - value: The value to include in a string interpolation, if non-`nil`.
+  ///   - default: The string to include if `value` is `nil`.
+  @backDeployed(before: SwiftStdlib 6.2)
+  @inlinable
```

Comment:
Any reason for `@backDeployed`+`@inlinable` instead of just `@_alwaysEmitIntoClient`?

Reply:
It seemed like the right set, presuming that `@inlinable` has different behavior than `@_alwaysEmit...`. Do we have guidance about which to prefer/when between inlinable and always-emit?

Reply:
These are very simple functions and I'm unsure if we want to commit these as "ABI". With AEIC, the stdlib doesn't take the ABI hit and clients who don't use it don't pay for it either.